data/examples/71--custom_fonts.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Custom Fonts",
+  "toc": "on",
+  "tags": [
+    "CSS",
+    "Custom Fonts"
+  ],
+  "layout": "dock-left",
+  "key": 71,
+  "code": "#code/eJyNkE1qwzAQRvc+xaBFaCixSJeO7ZYU2gO0F1DksSWQLeFRKKLk7pWUPxe6KGgzaL73PaYmHwy2BcCLHp2dPRxn88CU944qzns7eSoHaweDwmkqpR25JHp67sWoTWj2x2lAfPxQosNVp8kZERr6Eo6td0WiJsKmFxLhO44AlzmFK2AyHHDuwoRsl39pllU2iB3ncn5bKb3v12ntlMDlITdvKDX/yT6rQVZj99yN9w+hU1Hzy4GKWm1BGkHUsGU1az+VJojPK4T3fCl4i0yQwhjsYKlRc7Vtl6R73S+MsTGcza6U1+tiRvwAmrCP3A==",
+  "info": [
+    {
+      "title": "Custom Fonts in CSS",
+      "focus": null,
+      "text": "This example show show to use the <code>@import</code> and <code>@font-face</code> CSS rules to use custom fonts on your website from both a remote file and local file."
+    },
+    {
+      "title": "@import Rule",
+      "focus": [
+        2
+      ],
+      "text": "Here we are using the <code>@import</code> rule to load an external font from Google fonts. <a href=\"https://fonts.google.com/\" target=\"_blank\">Google Fonts</a> will give you code snippets that you can copy and paste into your CSS. In this example, we are importing the <href=\"https://fonts.google.com/specimen/Bungee+Shade?query=bungee\" target=\"_blank\">Bungee Shade</a> font by adding the link to the font to the <code>url</code> property. "
```

Comment:
instead of 
> by adding the link to the font to the <code>url</code> property.

how about 
> by passing the link to the CSS <code>url()</code> function.